bin/test_benchmarks

```diff
@@ -53,4 +57,43 @@ pushd test
     "
 popd || true
 
+# gather auto benchmark data for website
+
+# container size
+container_size=$(docker images \
+  | grep "cyberark\/secretless-broker[^-]" \
+  | awk '{ print $7 }')
+
+echo '<h3>Docker Image Size</h3>
+<p>The Docker image size is auto-updated on this page with every build of the
+project. You may also visit our
+<a href="https://hub.docker.com/r/cyberark/secretless-broker/">DockerHub</a>.</p>' \
+  | tee -a $project_dir/docs/_includes/auto_benchmarks.html
+echo "<p>Docker image size: ${container_size}</p>" \
+  | tee -a $project_dir/docs/_includes/auto_benchmarks.html
+
+echo "Container size: $container_size\n" \
+  | tee -a $project_dir/test/bench.output
+
+# lines of code count
+echo '<h2>Code Metrics</h2>
+<h3>Lines of Code</h3>
+<p>Code line counts were generated with
+<a href="https:github.com/hhatto/gocloc">gocloc</a> and are auto-updated
+with every build of this project.
+<pre>' \
+  | tee -a $project_dir/docs/_includes/auto_benchmarks.html
```

Comment:
this `tee` command should be a variable to reduce duplication. Also might be good to consider turning it into a function.

Reply:
Addendum: I feel like this is maybe ripe for having a template file and using that as the input for the html (or maybe just using markdown instead of pure html).